app/src/components/ReviewSection.tsx

```diff
@@ -0,0 +1,149 @@
+import { Trans } from 'next-i18next'
```

Comment:
Notes for reviewers: This component is broken out from `/review` so it can more easily be reused in `/confirmation`. In `/review` it is passed `true` for the `editable` prop.